modules/nf-core/hifiasm/tests/main.nf.test.snap

```diff
@@ -0,0 +1,150 @@
+{
+    "Test HifiAsm - fq [] [] fq fq": {
+        "content": [
+            [
+                [
+                    {
+                        "id": "test"
+                    },
+                    "test.asm.hic.r_utg.gfa:md5,9cb983397b093bcf9ef5a01c5aa52c6c"
+                ]
+            ],
+            [
+                
+            ],
+            [
+                [
+                    {
+                        "id": "test"
+                    },
+                    "test.asm.hic.p_utg.gfa:md5,3e2b4815145b06bb74f1b74876e767dc"
+                ]
+            ],
+            [
+                
+            ],
+            [
+                
+            ],
```

Comment:
optional outputs ?

Reply:
Yes, these are optional. Should I rather remove them from the test if we expect them to not be present?

Reply:
yes